### PR TITLE
Add <NativeLibrary /> documentation

### LINKED ISSUE #976: [NativeAot] Document DirectPInvoke's requirements
At first glance, i thought i can add any DllImport'ed library.
But then i added `"Ultralight"` and few more.
I received a lot of errors.
Beginning of errors
```
Создается библиотека bin\Release\net5.0\win-x64\native\UltralightNet.Veldrid.TestApp.lib и объект bin\Release\net5.0\win-x64\native\UltralightNet.Veldrid.TestApp.exp
UltralightNet.Veldrid.TestApp.obj : error LNK2001: неразрешенный внешний символ ulApplyProjection. [C:\Users\Дом\source\repos\SupinePandora43\UltralightSharp\UltralightNet.Veldrid.TestApp\UltralightNet.Veldrid.TestApp.csproj]
UltralightNet.Veldrid.TestApp.obj : error LNK2001: неразрешенный внешний символ ulCreateRenderer. [C:\Users\Дом\source\repos\SupinePandora43\UltralightSharp\UltralightNet.Veldrid.TestApp\UltralightNet.Veldrid.TestApp.csproj]
UltralightNet.Veldrid.TestApp.obj : error LNK2001: неразрешенный внешний символ ulDestroyRenderer. 
```

Sounds like it requires `lib` files.
I have them!
But where they should be?
How do i specify where It should search for them?!

Documentation doesn't answer these questions.
I tried looking at corert repo, but didn't found.
Could documentation be improved?

~~i tried asking this on discord (DotNetEvolution) but it wasn't answered~~

### PR DESCRIPTION
**Pull request by SupinePandora43**
fixes #976

### CODE REVIEW

File: docs/using-nativeaot/interop.md [hunk 1]
```diff
@@ -34,6 +34,21 @@ Examples:
 </ItemGroup>
 ```
 
+### Linking
+
+To statically link against a native library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file
```
Comment (jkotas) [inline docs/using-nativeaot/interop.md]:
```suggestion
To statically link against a unmanaged library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file on Windows and `.a` file on Unix
```

File: docs/using-nativeaot/interop.md [hunk 2]
```diff
@@ -34,6 +34,21 @@ Examples:
...
+
+Examples:
+
+```xml
+<ItemGroup>
+  <!-- Generate direct PInvokes for Dependency -->
+  <DirectPInvoke Include="Dependency" />
+  <!-- Specify .lib file -->
+  <NativeLibrary Include="Dependency.lib" />
```
Comment (jkotas) [inline docs/using-nativeaot/interop.md]:
```suggestion
  <!-- Specify library to link against -->
  <NativeLibrary Include="Dependency.lib" Condition="$(RuntimeIdentifier.StartsWith('win'))" />
  <NativeLibrary Include="Dependency.a" Condition="!$(RuntimeIdentifier.StartsWith('win'))" />
```

**Review by jkotas — APPROVED** [review]
Thank you!

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/runtimelab?pullRequest=980) <br/>All CLA requirements met.

**Comment by jkotas** [comment]
@SupinePandora43 Could you please sign the contributor license agreement?

File: docs/using-nativeaot/interop.md [hunk 3]
```diff
@@ -34,6 +34,22 @@ Examples:
 </ItemGroup>
 ```
 
+### Linking
+
+To statically link against a unmanaged library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file on Windows and `.a` file on Unix
```
Comment (Suchiman) [inline docs/using-nativeaot/interop.md]:
```suggestion
To statically link against a unmanaged library, you'll need to specify `<NativeLibrary Include="filename" />` pointing to `.lib` file on Windows and `.a` file on Unix
```

**Comment by SupinePandora43** [comment]
> @SupinePandora43 Could you please sign the contributor license agreement?

Done

**Pull request merged**